src/components/home/ExamplesSection.vue

```diff
@@ -189,14 +183,11 @@ const urls = swiperImagesUrls
 .examples-section__swiper {
```

Comment:
We need to create a separate slider.css/.scss file for all the slider classes, they will override the original swiper styles and can be reused for other sliders.

Reply:
For now leave like this, this is an ticket from last week that was not merged because of the tehnical problems with the main.
A new ticket will be created for this changes